paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -496,66 +480,60 @@ internal class PaymentSheetViewModelTest {
                 stripeIntent = it
             }
 
-            viewModel.onPaymentFlowResult(
-                PaymentFlowResult.Unvalidated()
-            )
+            viewModel.onPaymentResult(PaymentResult.Failed(Throwable()))
             verify(eventReporter)
                 .onPaymentFailure(selection)
 
             assertThat(stripeIntent).isNull()
         }
 
     @Test
-    fun `onPaymentFlowResult() with processing status for payment method which has delay should report success event`() =
+    fun `onPaymentResult() with processing status for payment method which has delay should report success event`() =
         testDispatcher.runBlockingTest {
-            whenever(paymentFlowResultProcessor.processResult(any())).thenReturn(
-                PaymentIntentResult(
-                    PaymentIntentFixtures.PI_WITH_SHIPPING.copy(
-                        paymentMethod = PaymentMethodFixtures.SEPA_DEBIT_PAYMENT_METHOD,
-                        status = StripeIntent.Status.Processing
-                    ),
-                    StripeIntentResult.Outcome.UNKNOWN
-                )
+            val paymentIntent = PaymentIntentFixtures.PI_WITH_SHIPPING.copy(
+                paymentMethod = PaymentMethodFixtures.SEPA_DEBIT_PAYMENT_METHOD,
+                status = StripeIntent.Status.Processing
+            )
+
+            val viewModel = createViewModel(
+                stripeIntentRepository = StripeIntentRepository.Static(paymentIntent),
             )
 
             val selection = PaymentSelection.Saved(PaymentMethodFixtures.SEPA_DEBIT_PAYMENT_METHOD)
             viewModel.updateSelection(selection)
 
-            viewModel.onPaymentFlowResult(
-                PaymentFlowResult.Unvalidated()
-            )
+            viewModel.onPaymentResult(PaymentResult.Completed)
+
             verify(eventReporter)
                 .onPaymentSuccess(selection)
         }
 
     @Test
-    fun `onPaymentFlowResult() with processing status for payment method which does not have delay should report failure event`() =
+    fun `onPaymentResult() with processing status for payment method which does not have delay should report failure event`() =
```

Comment:
similarly, this test could be deleted too

Reply:
done here: [fb76f58](https://github.com/stripe/stripe-android/pull/4159/commits/fb76f58e12f1724b9e51d0d7649b0db6e89bdf5a)